v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -132,6 +134,35 @@ def requires(self) -> ReferenceDataset:
             RAW_DATASET_PATH: 'https://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh38/clinvar.vcf.gz',
         },
     },
+    ReferenceDataset.splice_ai: {
+        ENUMS: {
+            'splice_consequence': [
+                'Acceptor gain',
+                'Acceptor loss',
+                'Donor gain',
+                'Donor loss',
+                'No consequence',
+            ],
+        },
+        SKIP_ENUM_ANNOTATION: True,
+        ReferenceGenome.GRCh37: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            RAW_DATASET_PATH: [
+                'gs://seqr-reference-data/GRCh37/spliceai/spliceai_scores.masked.snv.hg19.vcf.gz',
+                'gs://seqr-reference-data/GRCh37/spliceai/spliceai_scores.masked.indel.hg19.vcf.gz',
+            ],
+        },
+        ReferenceGenome.GRCh38: {
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            VERSION: '1.0',
+            # NB: SpliceAI data is only available to download for authenticated Illumina users, so we will host the data
+            RAW_DATASET_PATH: [
```

Comment:
I don't see these paths existing....


```
bblanken@wmbbe-67c seqr-loading-pipelines % gsutil ls gs://seqr-reference-data/GRCh37/spliceai/
gs://seqr-reference-data/GRCh37/spliceai/
gs://seqr-reference-data/GRCh37/spliceai/exome_spliceai_scores.vcf.gz
gs://seqr-reference-data/GRCh37/spliceai/whole_genome_filtered_spliceai_scores.vcf.gz
gs://seqr-reference-data/GRCh37/spliceai/spliceai_scores.ht/
bblanken@wmbbe-67c seqr-loading-pipelines % gsutil ls gs://seqr-reference-data/GRCh38/spliceai/
gs://seqr-reference-data/GRCh38/spliceai/
gs://seqr-reference-data/GRCh38/spliceai/exome_spliceai_scores.liftover.vcf.gz
gs://seqr-reference-data/GRCh38/spliceai/spliceai_scores.ht/
```

I also don't see the paths referenced in the original file!

Reply:
I asked Be about the originals and he told me it would be better to dowload them from Illumina directly rather than having someone try and go unearth the old ones